.github/workflows/build-and-test.yml

```diff
@@ -18,10 +18,12 @@ on:
     branches:
       - "master"
       - "branch-*"
+      - "main"
```

Comment:
why not just replace "master" with "main", instead of appending?

Reply:
I was thinking after we merge to master rename the branch to main and also add protection to the brach (so eventually master will be removed)